roles/jboss_fuse_on_karaf/tasks/main.yml

```diff
@@ -40,131 +45,189 @@
   # realpath to normalize paths.
 
   raw: locate karaf.jar | sed -n -e 's/\(.*\)lib\/karaf\.jar$/\1/p' | xargs -n 1 --no-run-if-empty readlink --canonicalize
-  register: karaf_locate_karaf_jar
+  register: internal_karaf_locate_karaf_jar
   ignore_errors: yes
   become: yes
   when: 'user_has_sudo and internal_have_locate and jboss_fuse'
 
+- name: set karaf_locate_karaf_jar fact 
+  set_fact:
+    karaf_locate_karaf_jar: '{{ internal_karaf_locate_karaf_jar }}'
+  ignore_errors: yes
+
 - name: Use find to look for karaf.jar
   raw: find {{search_directories}} -xdev -type f -name karaf.jar 2> /dev/null | sed -n -e 's/\(.*\)lib\/karaf\.jar$/\1/p' | xargs -n 1 --no-run-if-empty readlink --canonicalize | sort -u
-  register: karaf_find_karaf_jar
+  register: internal_karaf_find_karaf_jar
   ignore_errors: yes
   become: yes
   when: 'user_has_sudo and jboss_fuse_ext'
 
+- name: set karaf_find_karaf_jar fact 
+  set_fact:
+    karaf_find_karaf_jar: '{{ internal_karaf_find_karaf_jar }}'
+  ignore_errors: yes
+
 # Combine the outputs of the above into a single fact
 
 - name: combine KARAF_HOME candidates into single list
   set_fact:
       karaf_homes: "{{
-          (karaf_running_processes.get('stdout_lines', []) +
-            karaf_locate_karaf_jar.get('stdout_lines', []) +
-            karaf_find_karaf_jar.get('stdout_lines', []))|difference(['']) | unique }}"
+          (internal_karaf_running_processes.get('stdout_lines', []) +
+            internal_karaf_locate_karaf_jar.get('stdout_lines', []) +
```

Comment:
it could be the way the diff is showing up but should the second two variables indention be lined up underneath the first?

Reply:
🤕

Reply:
😬 sry